pandas/core/window.py

```diff
@@ -963,8 +964,19 @@ def count(self):
         not passed. In the future `raw` will default to False.
 
         .. versionadded:: 0.23.0
+    *args, **kwargs :
```

Comment:
Would this make sense in `_doc_template` as well? Pretty generic so if it applies to all types of aggregation better placed there

Reply:
Cool, will look into it

Reply:
@WillAyd I've looked into how adding ``` *args, **kwargs``` to ```_doc_template``` will affect the methods with the ```@Appender(_doc_template)``` decorator. 

Adding ``` *args, **kwargs``` makes sense for some of the method such as ```pandas.core.window.Rolling.max```, where it takes in ``` *args, **kwargs``` as parameters :
```
    @Substitution(name='rolling')
    @Appender(_doc_template)
    @Appender(_shared_docs['max'])
    def max(self, *args, **kwargs):
        nv.validate_rolling_func('max', args, kwargs)
        return super(Rolling, self).max(*args, **kwargs)
```

However adding ``` *args, **kwargs``` does not fully make sense for ```pandas.core.window.Expanding.skew``` as it does not have an ```*args``` parameter:
```
 @Substitution(name='expanding')
    @Appender(_doc_template)
    @Appender(_shared_docs['skew'])
    def skew(self, **kwargs):
        return super(Expanding, self).skew(**kwargs)
```


I don't think adding the ``` *args, **kwargs``` should be done in the ```_doc_template``` but it should rather be done in each method's respective ```_shared_docs``` to make sure that the docstring truly reflects what the function does and expects as parameters.  We could maybe spawn new issues to address these ```_shared_docs``` updates?

What do you advise though? @WillAyd & @datapythonista